tests_integration/__tests__/check.js

```diff
@@ -7,7 +7,8 @@ describe("checks stdin with --check", () => {
     input: "0"
   }).test({
     stdout: "(stdin)\n",
-    stderr: "",
+    stderr:
+      "[warn] --parser=babylon is deprecated; we now treat it as --parser=babel.\n",
```

Comment:
How should I test deprecation warnings?

Reply:
It's okay to test it this way but you should add another test case instead of using existing tests. (Different purpose should use different `test` with appropriate test name.)

Reply:
Oh I see. I can add another test under integration tests that explicitly tests for warnings